tempodb/backend/cache/cache.go

```diff
@@ -1,45 +1,30 @@
-package redis
+package cache
 
 import (
 	"context"
 	"io"
-	"time"
-
-	"github.com/cortexproject/cortex/pkg/chunk/cache"
-	"github.com/go-kit/kit/log"
-	"github.com/grafana/tempo/tempodb/backend"
 
 	"github.com/google/uuid"
+	"github.com/grafana/tempo/tempodb/backend"
 )
 
-type Config struct {
-	ClientConfig cache.RedisConfig `yaml:",inline"`
-
-	TTL time.Duration `yaml:"ttl"`
-}
-
 type readerWriter struct {
 	nextReader backend.Reader
 	nextWriter backend.Writer
-	client     *cache.RedisCache
-	logger     log.Logger
+	cache      Client
```

Comment:
This is a little confusing, let's make this `client    Client`

Reply:
Agree! Changed :)